src/app/screens/Settings.tsx

```diff
@@ -133,8 +133,18 @@ function Settings() {
               label={t("camera_access.allow")}
               onClick={async () => {
```

Comment:
Can we move this click handler to a separate function? This would improve readability for me and - although it might not be critical here - it's a good idea in general for rendering performance: https://marcoheine.com/blog/7-easy-ways-to-improve-your-react-performance-part-2/#avoid-anonymous-funcs